product-service/.serverless/cloudformation-template-create-stack.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Looks like files under `.serverless` directory better to add to `.gitignore` 🙂

Reply:
Good idea!) Thank you!

Reply:
Fixed